tests/integration/dev.bats

```diff
@@ -144,3 +144,22 @@ function teardown() {
 	runc exec -t test_exec sh -c "ls -l /proc/self/fd/0; echo 123"
 	[ "$status" -eq 0 ]
 }
+
+@test "runc run [hostname domainname]" {
```

Comment:
nit: why are you putting it to `dev.bats` -- this file has various device tests, while your test does not have anything to do with devices.

You can use something generic like `run.bats` instead.

Reply:
Sorry, I missed `run.bats`